### PR TITLE
Final revised ticket categories

### PR DESCRIPTION
**Pull request by mtmueller**

### CODE REVIEW

File: _zendesk_block.view.lkml [hunk 1]
```diff
@@ -425,37 +425,33 @@ view: ticket {
...
-        label: "Onboard"
-      }
-      when: {
-        sql:${custom_ticket_categories} IN("data_error", "feature_request/_product_feedback", "platform_downtime", "publish__bug__p2s2", "publish__bug__p2s3", "publish__bug__p3s2" ,"publish__bug__p3s3","data_request", "publish__other", "publish__unknown","training_gap", "publish__bug__p1s2", "publish__bug__p2s1", "publish__bug__p1s3", "publish__bug__p1s1", "publish__bug__p4s3", "publish__bug__p3s4");;
+        sql:${custom_ticket_categories} like "%publish%" IN("data_error", "feature_request/_product_feedback", "data_request", "training_gap", "platform_downtime");;
```
Comment (charliechaf) [inline _zendesk_block.view.lkml]: There is an error in this SQL

File: _zendesk_block.view.lkml [hunk 2]
```diff
@@ -74,13 +74,13 @@ view: ticket {
       label: "Capture Bug S1-Blocker"
       }
       when: {
-        sql: ${custom_ticket_categories}= "capture__bug__p1s2";;
+        sql: ${custom_ticket_categories}= "capture__bug__p1s2" AND ${custom_ticket_categories}= "capture__bug__p2s2" AND ${custom_ticket_categories}= "capture__bug__p3s2";;
```
Comment (charliechaf) [inline _zendesk_block.view.lkml]: the `AND` in this SQL and all the other CASES below need to be changed to `OR`

File: _zendesk_block.view.lkml (hunk 1 above)
Comment (charliechaf) [inline _zendesk_block.view.lkml]:
```suggestion
        sql:${custom_ticket_categories} like "%publish%" OR ${custom_ticket_categories} IN("data_error", "feature_request/_product_feedback", "data_request", "training_gap", "platform_downtime");;
```

**Comment by mtmueller** [comment]
Hell ya

**Pull request merged**